tilestache.cfg.sample

```diff
@@ -38,6 +38,38 @@
                 "name": "mbtiles",
                 "tileset": "data/mbtiles/markernoven.mbtiles"
             }
+        },
+        "productsnogeopath":
+        {
+            "provider":
+            {
+                "name": "mbtiles",
+                "tileset": "data/products-nogeopath.mbtiles"
```

Comment:
Would we want to, by default, put these in a subdirectory similar to the ones above defaulting to `data/mbtiles`?

Reply:
Nice catch. These should have the same structure.